pyproject.toml

```diff
@@ -80,6 +80,7 @@ hypothesis = "^6.23.0"
 Sphinx = "^4.2.0"
 sphinx-bootstrap-theme = "^0.8.0"
 typing-extensions = ">=3.7.4"
+pytest-randomly = "^3.10.1"
```

Comment:
Thats it? We dont need to do anything else? :-D

Reply:
once you have it as a plugin, `pytest` will pick up that it's available and it'll just automatically be used. If we want we can control the random seed to fix the order of the tests (so that we can reproduce), but I don't think that's necessary.